src/server/svr_jobfunc.c

```diff
@@ -4929,36 +4916,32 @@ determine_accruetype(job* pjob)
 	temphold = pjob->ji_wattr[(int)JOB_ATR_hold].at_val.at_long;
 	if (temphold & HOLD_o || temphold & HOLD_bad_password || temphold & HOLD_s) {
 		if ((pjob->ji_qs.ji_substate == JOB_SUBSTATE_DEPNHOLD)
-			&& (temphold & HOLD_s)) {
-			newaccruetype = (long)JOB_INELIGIBLE;
-		} else {
-			newaccruetype = (long)JOB_ELIGIBLE;
-		}
-		return newaccruetype;
+			&& (temphold & HOLD_s))
+			return JOB_INELIGIBLE;
+		else
+			return JOB_ELIGIBLE;
 	}
 
 	/* scheduler suspend job ; accrue eligible time */
-	if (pjob->ji_qs.ji_substate == JOB_SUBSTATE_SCHSUSP) {
-		newaccruetype = (long)JOB_ELIGIBLE;
-		return newaccruetype;
-	}
+	if (pjob->ji_qs.ji_substate == JOB_SUBSTATE_SCHSUSP)
+		return JOB_ELIGIBLE;
 
 	/* qsig suspended job ; accrue eligible time */
-	if (pjob->ji_qs.ji_substate == JOB_SUBSTATE_SUSPEND) {
-		newaccruetype = (long)JOB_ELIGIBLE;
-		return newaccruetype;
-	}
+	if (pjob->ji_qs.ji_substate == JOB_SUBSTATE_SUSPEND)
+		return JOB_ELIGIBLE;
 
 	/* check for stopped queue: routing and execute ; accrue eligible time */
 	pque = find_queuebyname(pjob->ji_qs.ji_queue);
 	if (pque != NULL)
-		if (pque->qu_attr[(int)QA_ATR_Started].at_val.at_long == 0) {
-			newaccruetype = (long)JOB_ELIGIBLE;
-			return newaccruetype;
-		}
+		if (pque->qu_attr[(int)QA_ATR_Started].at_val.at_long == 0)
+			return JOB_ELIGIBLE;
+
+	/* New job without any apparent crimes, make it accrue eligible time */
```

Comment:
Did you mean times instead of crimes?

Reply:
no i did mean crimes :) I meant to say that this job doesn't seem like it has a hold, or has anything else that'd prevent it from accruing eligible time .. would you like me to word it differently?

Reply:
Yeah, I was a little confused on the wording.  Maybe something like "The job doesn't have any reason to not accrue eligible time (e.g. on hold), so it should accrue it". Or something like that